security/4.0/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Jakarta Security 4.0 (under development)"
+date: 2023-05-30
+summary: "Release for Jakarta EE 11"
+---
+Jakarta Security defines a standard for creating secure Jakarta EE applications in modern application paradigms.
+
+The release splits focuses on evolving the API in various ways, and specifically providing APIs for the authorization theme (including interceptors and an abstraction for the permission store)
+
+### New features, enhancements or additions
+* Client-cert and Digest SECURITY #120 
+* Authentication mechanism per URL SECURIY #86 
+* User choice of authentication mechanism (login with provider X, login with provider Y, etc) 
+* Multiple authentication mechanisms (try JWT, fallback to BASIC, etc) 
+* @RolesAllowed alternative 
+* Easily adding an interceptor to a build-in CDI bean
+* Authorization modules
+
+### Removals, deprecations or backwards incompatible changes
+* Remove references to the SecurityManager
```

Comment:
Hi @arjantijms ,
As per checklist template [1]. Do you have a link or reference that describes in more detail the overall removal of the Security Manager?

[1]
> Includes a detailed description of any backward incompatibility

Reply:
The SecurityManager is not used explicitly by Jakarta Security. This concern is delegated to Jakarta Authorization, this was just a blanket statement really. The APIs may contain SecurityManager references internally, just like some other APIs (not related to security themselves do).

See: https://github.com/search?q=repo%3Ajakartaee%2Fsecurity%20SecurityManager&type=code